utils/server_handler.py

```diff
@@ -0,0 +1,125 @@
+import http.client
+import subprocess
+import os
+
+DEFAULT_SERVER_LOCATION = "C:/Users/Matt/Dev/CS52-Autograder/serve-sml"
+DEFAULT_FILE_ORDER = ["buffer", "util", "http", "server", "assert", "grading_lib", "main"]
+
+class ServerHandler:
+
+    def __init__(self, server_location:str=None, files:list=None):
```

Comment:
sadly the style guides suggest this be `__init__(self, server_location: str = None, files : list = None)`
Except also your server location should probably be a `Path` type, not a `str` type. And given it can be `None`, it should probably be an `Optional[Path]` with `Optional` imported from `typing`
Similar `files` should probably be of type `Optional[List[Path]]`